pkg/services/service_item/service_item_fetcher.go

```
@@ -0,0 +1,22 @@
+package serviceitem
+
+import (
+	"github.com/transcom/mymove/pkg/appcontext"
+	"github.com/transcom/mymove/pkg/models"
+	"github.com/transcom/mymove/pkg/services"
+)
+
+type serviceItemFetcher struct {
+}
+
+// NewPaymentRequestFetcher returns a new payment request fetcher
+func NewServiceItemFetcher() services.ServiceItemListFetcher {
+	return &serviceItemFetcher{}
+}
+
+func (s *serviceItemFetcher) FetchServiceItemList(appCtx appcontext.AppContext) (*models.ReServiceItems, error) {
+
+	var serviceItems models.ReServiceItems
+	err := appCtx.DB().Eager("ReService").All(&serviceItems)
```

You need to return an error here before the return:
```
if err != nil
```

---

I added an error check.